src/components/actionBar/_utils/actionIconSchema.tsx

```diff
@@ -21,7 +20,7 @@ export interface ActionIconSchema {
   tooltip: {
     message: string;
   };
-  form: (modalState: ActiveIndexStateMethods) => React.ReactNode;
+  form: (onClose: () => void) => React.ReactNode;
```

Comment:
Is there a way to refer to where this is declared, so I don't have to change all these references if it takes an argument or something eventually.

Reply:
Naw, Ill do this when its an issue